scripts/secnetperf.ps1

```diff
@@ -254,11 +254,7 @@ if ($isWindows) {
 $allTests = [System.Collections.Specialized.OrderedDictionary]::new()
 
 # > All tests:
-$allTests["tput-up"] = "-exec:maxtput -up:12s -ptput:1"
-$allTests["tput-down"] = "-exec:maxtput -down:12s -ptput:1"
-$allTests["hps-conns-100"] = "-exec:maxtput -rconn:1 -share:1 -conns:100 -run:12s -prate:1"
-$allTests["rps-up-512-down-4000"] = "-exec:lowlat -rstream:1 -up:512 -down:4000 -run:20s -plat:1"
-$allTests["max-rps-up-512-down-4000"] = "-exec:lowlat -conns:16cpu -streams:10 -rstream:1 -up:512 -down:4000 -run:20s -plat:1"
+$allTests = @("upload", "download", "hps", "rps-single", "rps-multi", "latency")
```

Comment:
I think this is effectively a breaking change, from an identifier `tput-up` to `upload` (and so on), and I might need to keep the old ID and map it to the scenario. @ProjectsByJackHe please let me know what you think.

Reply:
@ProjectsByJackHe as we discussed in our meeting yesterday, we need to ensure we're Ok with this change, and then update the dashboard and other netperf files as necessary. Please take a look. Thanks!

Reply:
Tracking: https://github.com/microsoft/netperf/issues/401